scripts/tasks.sh

```diff
@@ -19,7 +19,7 @@ DIR=.
 . $DIR/scripts/distro/interface.sh
 . $DIR/scripts/distro/kubeadm/distro.sh
 . $DIR/scripts/distro/rke2/distro.sh
-. $DIR/scripts/distro/rke2/addon.sh
```

Comment:
why are you removing this?

Reply:
because I added it 20 minutes ago and meant to add this instead

Reply:
`distro/rke2/addon.sh` doesn't even exist

Reply:
ok